examples/contact-us-form/main.bal

```diff
@@ -0,0 +1,218 @@
+                }
+            },
+            legalConsentOptions: {
+                'type: "none"
+            }
+        };
+
+
+    
+    forms:FormDefinitionBase response = check baseClient->/.post(
+        inputFormDefinition     
+    );
+    string formId = response?.id;
+    io:println("Form is created  with ID:  " + formId);
+
+
+
+    forms:FormDefinitionBase updateResponse = check baseClient->/[formId].patch(
+        {
+            fieldGroups: [
+                {
+                    groupType: "default_group",
+                    richTextType: "text",
+                    fields: [
+                        {
+                            objectTypeId: "0-1",
+                            name: "email",
+                            label: "Email",
+                            required: true,
+                            hidden: false,
+                            fieldType: "email",
+                            validation: {
+                                blockedEmailDomains: [],
+                                useDefaultBlockList: false
+                            }
+                               
+                        }
+                    ]
+                },
+                 {
+      groupType: "default_group",
+      richTextType: "text",
+      fields: [
+        {
+          objectTypeId: "0-1",
+          name: "firstname",
+          label: "First name",
+          required: true,
+          hidden: false,
+          fieldType: "single_line_text"
+        },
+         {
+          objectTypeId: "0-1",
+          name: "lastname",
+          label: "Last name",
+          required: true,
+          hidden: false,
+          fieldType: "single_line_text"
+        }
+      ]
+    },
+      {
+      groupType: "default_group",
+      richTextType: "text",
+      fields: [
+        {
+          objectTypeId: "0-1",
+          name: "phone",
+          label: "Phone Number",
+          required: false,
+          hidden: false,
+          fieldType: "phone",
+          useCountryCodeSelect:true,
+          validation : {
+            minAllowedDigits: 10,
+            maxAllowedDigits: 10
+
+            
+          }
+          
+        }
+      ]
+    },
+    {
+
+         groupType: "default_group",
+      richTextType: "text",
+        fields: [
+            {
+            objectTypeId: "0-1",
+            name: "message",
+            label: "Message",
+            required: true,
+            hidden: false,
+            fieldType: "multi_line_text"
+            }
+        ]
+
+    }
+            ]
+        }
+    );
+
+    io:println("Form is updated at" + updateResponse?.updatedAt);
+
+
+    forms:FormDefinitionBase getResponse = check baseClient->/[formId]();
+    io:println("Form is created at" + getResponse?.createdAt);
+
+
+
+
+
+
+    json deleteResponse = check baseClient->/[formId].delete();
+
+    if (deleteResponse == null){
+        io:println("Form is deleted");}
+
+
+   
+
+
+
+    
```

Comment:
Since this returns nil, you don't have to assign a value from the return. Also please check the extra empty lines and remove them.

```suggestion
    check baseClient->/[formId].delete();
    io:println("Form is deleted");
```